uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/MsalTestApp.java

```diff
@@ -172,6 +186,7 @@ public String getActiveBrokerPackageName() throws UiObjectNotFoundException {
         return text;
     }
 
+    // check MsalTestApp mode
     public String checkMode() throws UiObjectNotFoundException {
```

Comment:
I noticed that some of the tests in which checkMode() is getting called are failing. This is because, it takes some time for the screen to load for checkMode to be reflected. Could you please add a sleep (3secs should be enough) everywhere this method is being called?

Reply:
this method is only used in TestCase2582292. I added wait time after hanleFirstRun here:
![image](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/assets/51412053/a37ce600-c357-43b6-8d81-eccf95ac106d)